### ISSUE TITLE
Babel throws error

### ISSUE THREAD

**Issue by mastertinner** [fraserxu/react-dropdown#35]
Im trying to use this component with `babel` (`babel-core 6.5.2 to be specific`). This works fine with version `react-dropdown 0.7.2` but since version `1.0.0`, I am getting the following error:
```
Module parse failed: .../node_modules/react-dropdown/index.js Line 1: Unexpected token
```

**Comment by fraserxu** [comment]
Do you have the proper babel preset in your setup?

**Comment by mastertinner** [comment]
@fraserxu, thanks for your reply. This is my `.babelrc`:
```json
{
  "presets": [
    "es2015",
    "react"
  ],
  plugins: [
    [
      "typecheck",
      {
        "disable": {
          "production": true
        }
      }
    ]
  ]
}
```

**Comment by fraserxu** [comment]
Can you try 

```
{
  "presets": ["es2015", "stage-0", "react"]
}
```

?

**Comment by fraserxu** [comment]
Sorry @mastertinner I noticed I forget to push the `dist` directory to npm, you don't need anything to use it. And I just pushed the fix to v`1.0.2`. Can you try update the version?

**Comment by mastertinner** [comment]
that fixed it. thanks so much for the quick fix, @fraserxu!
